test/compliance/resources/db/basic/literals/basic.literals.string.js

```diff
@@ -31,6 +31,9 @@ module.exports = [
   {
     char: 'A',
   },
+  {
+    char: '대', // Ensure multi byte utf-8 characters also fit into a single character column
```

Comment:
Are we sure, that the character that is not utf-8 converted is still the same, when read from database?
Maybe a test can be added, that INSERTed value is the same then the afterwards selected value...